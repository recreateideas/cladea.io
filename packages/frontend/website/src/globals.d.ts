import { Theme } from "./App/ui-core/theme";
import { Store } from "redux";

export declare global {
  interface Window {
    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: any;
    /** enzyme global methods for testing */
    shallow: any;
    render: any;
    mount: any;
    theme: Theme;
    ResizeObserver: any;
    store: Store;
  }
}

declare module "ui-core";
