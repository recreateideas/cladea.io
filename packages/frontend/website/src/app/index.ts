export * from "./App";
export * from "./features";
