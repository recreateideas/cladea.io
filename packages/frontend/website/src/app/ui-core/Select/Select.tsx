import { ReactElement, forwardRef, Ref } from 'react';
import { TextField, StandardTextFieldProps, MenuItem } from '@mui/material';
import { Container } from './styledComponents';

interface IProps extends StandardTextFieldProps {
    options: Array<{
        value: string | number;
        label: string;
    }>;
}
export const Select = forwardRef((props: IProps, ref: Ref<any>): ReactElement => {
    const { options, variant = 'standard', ...textFieldProps } = props;
    return (
        <Container className="select-container">
            <TextField ref={ref} select {...{ ...textFieldProps, variant }}>
                {options.map(({ value: optionValue, label: optionLabel }, index) => (
                    <MenuItem key={index} value={optionValue}>
                        {optionLabel}
                    </MenuItem>
                ))}
            </TextField>
        </Container>
    );
});

Select.displayName = 'Select';

Select.defaultProps = {};
