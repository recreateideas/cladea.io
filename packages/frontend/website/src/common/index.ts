export * from './utils';
export * from './hooks';
export * from './converters';
