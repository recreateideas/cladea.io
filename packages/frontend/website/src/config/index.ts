export const apiGatewaUrl = process.env.REACT_APP_API_GATEWAY_URL;

console.log(process.env);
