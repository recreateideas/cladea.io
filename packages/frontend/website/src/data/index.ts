import land110m from "./land-110m.json";

export { land110m };
