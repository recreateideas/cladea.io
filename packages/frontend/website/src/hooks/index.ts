export * from "./ClickOutside";
export * from "./use-scroll-y";
export * from "./use-margin";
export * from "./use-breakpoints";
