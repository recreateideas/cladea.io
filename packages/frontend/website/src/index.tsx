import "./app/ui-core/theme/GlobalStyle/roboto.css";
import ReactDOM from "react-dom";
import { Provider } from "react-redux";
import { store } from "./redux";
import { ServicedApp, ThemeProviderConnected } from "./app";

ReactDOM.render(
  <Provider store={store}>
    <ThemeProviderConnected>
      <ServicedApp />
    </ThemeProviderConnected>
  </Provider>,
  document.getElementById("root")
);
