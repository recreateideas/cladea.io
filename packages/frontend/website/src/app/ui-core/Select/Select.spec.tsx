import { Select } from './Select';
import { TestProvider } from 'src/tests';

const { mountWithProviderAndTheme } = TestProvider();
describe('<Select/>', () => {
    it('should render', () => {
        const state = {
            common: {},
        };
        const root = mountWithProviderAndTheme(<Select value="" options={[]} />, state);
        expect(root).toMatchSnapshot();
    });

    it('should render all the options', () => {
        const state = {
            common: {},
        };
        const root = mountWithProviderAndTheme(
            <Select
                value="chocolate"
                options={[
                    { value: 'chocolate', label: 'Chocolate' },
                    { value: 'strawberry', label: 'Strawberry' },
                    { value: 'vanilla', label: 'Vanilla' },
                ]}
            />,
            state
        );
        expect(root).toMatchSnapshot();
    });
});
