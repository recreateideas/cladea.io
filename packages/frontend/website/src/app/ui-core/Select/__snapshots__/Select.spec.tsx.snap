// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select/> should render 1`] = `
Array [
  .c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
}

.c0 {
  width: 100%;
}

@media (hover:none) {

}

<div
    className="c0 select-container"
  >
    <div
      className="c1 MuiFormControl-root MuiTextField-root-gGZDs MuiTextField-root"
    >
      Array [
        "",
        .c1 {
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.4375em;
  -webkit-letter-spacing: 0.00938em;
  -moz-letter-spacing: 0.00938em;
  -ms-letter-spacing: 0.00938em;
  letter-spacing: 0.00938em;
  color: #031A51;
  box-sizing: border-box;
  position: relative;
  cursor: text;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1.Mui-disabled {
  color: rgba(0,0,0,0.38);
  cursor: default;
}

.c2 {
  position: relative;
}

label + .c0 {
  margin-top: 16px;
}

.c2:after {
  border-bottom: 2px solid #12D8F7;
  left: 0;
  bottom: 0;
  content: "";
  position: absolute;
  right: 0;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 200ms cubic-bezier(0.0,0,0.2,1) 0ms;
  -webkit-transition: transform 200ms cubic-bezier(0.0,0,0.2,1) 0ms;
  transition: transform 200ms cubic-bezier(0.0,0,0.2,1) 0ms;
  pointer-events: none;
}

.c2.Mui-focused:after {
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2.Mui-error:after {
  border-bottom-color: #d32f2f;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2:before {
  border-bottom: 1px solid rgba(0,0,0,0.42);
  left: 0;
  bottom: 0;
  content: "\\00a0";
  position: absolute;
  right: 0;
  -webkit-transition: border-bottom-color 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: border-bottom-color 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  pointer-events: none;
}

.c2:hover:not(.Mui-disabled):before {
  border-bottom: 2px solid #031A51;
}

.c2.Mui-disabled:before {
  border-bottom-style: dotted;
}

@media (hover:none) {
  .c2:hover:not(.Mui-disabled):before {
    border-bottom: 1px solid rgba(0,0,0,0.42);
  }
}

<div
          className="c0 c1 c2 MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiSelect-root"
          onClick={[Function]}
        >
          Array [
            .c2 {
  font: inherit;
  -webkit-letter-spacing: inherit;
  -moz-letter-spacing: inherit;
  -ms-letter-spacing: inherit;
  letter-spacing: inherit;
  color: currentColor;
  padding: 4px 0 5px;
  border: 0;
  box-sizing: content-box;
  background: none;
  height: 1.4375em;
  margin: 0;
  -webkit-tap-highlight-color: transparent;
  display: block;
  min-width: 0;
  width: 100%;
  -webkit-animation-name: mui-auto-fill-cancel;
  animation-name: mui-auto-fill-cancel;
  -webkit-animation-duration: 10ms;
  animation-duration: 10ms;
}

.c2::-webkit-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2::-moz-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2:-ms-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2::-ms-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2:focus {
  outline: 0;
}

.c2:invalid {
  box-shadow: none;
}

.c2::-webkit-search-decoration {
  -webkit-appearance: none;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1::-webkit-input-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1::-moz-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:-ms-input-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1::-ms-input-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus::-webkit-input-placeholder {
  opacity: 0.42;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus::-moz-placeholder {
  opacity: 0.42;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus:-ms-input-placeholder {
  opacity: 0.42;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus::-ms-input-placeholder {
  opacity: 0.42;
}

.c2.Mui-disabled {
  opacity: 1;
  -webkit-text-fill-color: rgba(0,0,0,0.38);
}

.c2:-webkit-autofill {
  -webkit-animation-duration: 5000s;
  animation-duration: 5000s;
  -webkit-animation-name: mui-auto-fill;
  animation-name: mui-auto-fill;
}

.c0 {
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 0;
  cursor: pointer;
}

.c0:focus {
  background-color: rgba(0,0,0,0.05);
  border-radius: 0;
}

.c0::-ms-expand {
  display: none;
}

.c0.Mui-disabled {
  cursor: default;
}

.c0[multiple] {
  height: auto;
}

.c0:not([multiple]) option,
.c0:not([multiple]) optgroup {
  background-color: #fff;
}

.c0.c0.c0 {
  padding-right: 24px;
  min-width: 16px;
}

.c0.MuiSelect-select {
  height: auto;
  min-height: 1.4375em;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}

@media (hover:none) {

}

<div
              aria-expanded="false"
              aria-haspopup="listbox"
              className="c0 MuiSelect-select MuiSelect-standard c1 MuiInput-input-tFIHl c2 MuiInput-input MuiInputBase-input"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseDown={[Function]}
              role="button"
              tabIndex={0}
            >
              <span
                className="notranslate"
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </div>,
            .c0 {
  bottom: 0;
  left: 0;
  position: absolute;
  opacity: 0;
  pointer-events: none;
  width: 100%;
  box-sizing: border-box;
}

@media (hover:none) {

}

<input
              aria-hidden={true}
              autoFocus={false}
              className="c0 MuiSelect-nativeInput"
              disabled={false}
              onAnimationStart={[Function]}
              onChange={[Function]}
              required={false}
              tabIndex={-1}
              value=""
            />,
            .c0 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: fill 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  font-size: 1.5rem;
}

.c1 {
  position: absolute;
  right: 0;
  top: calc(50% - .5em);
  pointer-events: none;
  color: rgba(0,0,0,0.54);
}

.c1.Mui-disabled {
  color: rgba(0,0,0,0.26);
}

@media (hover:none) {

}

<svg
              aria-hidden={true}
              className="c0 MuiSvgIcon-root MuiSvgIcon-fontSizeMedium c1 MuiSelect-icon MuiSelect-iconStandard"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>,
            "",
          ]
        </div>,
      ]
    </div>
  </div>,
  "",
]
`;

exports[`<Select/> should render all the options 1`] = `
Array [
  .c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
}

.c0 {
  width: 100%;
}

@media (hover:none) {

}

<div
    className="c0 select-container"
  >
    <div
      className="c1 MuiFormControl-root MuiTextField-root-gGZDs MuiTextField-root"
    >
      Array [
        "",
        .c1 {
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.4375em;
  -webkit-letter-spacing: 0.00938em;
  -moz-letter-spacing: 0.00938em;
  -ms-letter-spacing: 0.00938em;
  letter-spacing: 0.00938em;
  color: #031A51;
  box-sizing: border-box;
  position: relative;
  cursor: text;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1.Mui-disabled {
  color: rgba(0,0,0,0.38);
  cursor: default;
}

.c2 {
  position: relative;
}

label + .c0 {
  margin-top: 16px;
}

.c2:after {
  border-bottom: 2px solid #12D8F7;
  left: 0;
  bottom: 0;
  content: "";
  position: absolute;
  right: 0;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 200ms cubic-bezier(0.0,0,0.2,1) 0ms;
  -webkit-transition: transform 200ms cubic-bezier(0.0,0,0.2,1) 0ms;
  transition: transform 200ms cubic-bezier(0.0,0,0.2,1) 0ms;
  pointer-events: none;
}

.c2.Mui-focused:after {
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2.Mui-error:after {
  border-bottom-color: #d32f2f;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2:before {
  border-bottom: 1px solid rgba(0,0,0,0.42);
  left: 0;
  bottom: 0;
  content: "\\00a0";
  position: absolute;
  right: 0;
  -webkit-transition: border-bottom-color 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: border-bottom-color 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  pointer-events: none;
}

.c2:hover:not(.Mui-disabled):before {
  border-bottom: 2px solid #031A51;
}

.c2.Mui-disabled:before {
  border-bottom-style: dotted;
}

@media (hover:none) {
  .c2:hover:not(.Mui-disabled):before {
    border-bottom: 1px solid rgba(0,0,0,0.42);
  }
}

<div
          className="c0 c1 c2 MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiSelect-root"
          onClick={[Function]}
        >
          Array [
            .c2 {
  font: inherit;
  -webkit-letter-spacing: inherit;
  -moz-letter-spacing: inherit;
  -ms-letter-spacing: inherit;
  letter-spacing: inherit;
  color: currentColor;
  padding: 4px 0 5px;
  border: 0;
  box-sizing: content-box;
  background: none;
  height: 1.4375em;
  margin: 0;
  -webkit-tap-highlight-color: transparent;
  display: block;
  min-width: 0;
  width: 100%;
  -webkit-animation-name: mui-auto-fill-cancel;
  animation-name: mui-auto-fill-cancel;
  -webkit-animation-duration: 10ms;
  animation-duration: 10ms;
}

.c2::-webkit-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2::-moz-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2:-ms-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2::-ms-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
}

.c2:focus {
  outline: 0;
}

.c2:invalid {
  box-shadow: none;
}

.c2::-webkit-search-decoration {
  -webkit-appearance: none;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1::-webkit-input-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1::-moz-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:-ms-input-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1::-ms-input-placeholder {
  opacity: 0 !important;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus::-webkit-input-placeholder {
  opacity: 0.42;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus::-moz-placeholder {
  opacity: 0.42;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus:-ms-input-placeholder {
  opacity: 0.42;
}

label[data-shrink=false] + .MuiInputBase-formControl .c1:focus::-ms-input-placeholder {
  opacity: 0.42;
}

.c2.Mui-disabled {
  opacity: 1;
  -webkit-text-fill-color: rgba(0,0,0,0.38);
}

.c2:-webkit-autofill {
  -webkit-animation-duration: 5000s;
  animation-duration: 5000s;
  -webkit-animation-name: mui-auto-fill;
  animation-name: mui-auto-fill;
}

.c0 {
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 0;
  cursor: pointer;
}

.c0:focus {
  background-color: rgba(0,0,0,0.05);
  border-radius: 0;
}

.c0::-ms-expand {
  display: none;
}

.c0.Mui-disabled {
  cursor: default;
}

.c0[multiple] {
  height: auto;
}

.c0:not([multiple]) option,
.c0:not([multiple]) optgroup {
  background-color: #fff;
}

.c0.c0.c0 {
  padding-right: 24px;
  min-width: 16px;
}

.c0.MuiSelect-select {
  height: auto;
  min-height: 1.4375em;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}

@media (hover:none) {

}

<div
              aria-expanded="false"
              aria-haspopup="listbox"
              className="c0 MuiSelect-select MuiSelect-standard c1 MuiInput-input-tFIHl c2 MuiInput-input MuiInputBase-input"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseDown={[Function]}
              role="button"
              tabIndex={0}
            >
              Chocolate
            </div>,
            .c0 {
  bottom: 0;
  left: 0;
  position: absolute;
  opacity: 0;
  pointer-events: none;
  width: 100%;
  box-sizing: border-box;
}

@media (hover:none) {

}

<input
              aria-hidden={true}
              autoFocus={false}
              className="c0 MuiSelect-nativeInput"
              disabled={false}
              onAnimationStart={[Function]}
              onChange={[Function]}
              required={false}
              tabIndex={-1}
              value="chocolate"
            />,
            .c0 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: fill 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  font-size: 1.5rem;
}

.c1 {
  position: absolute;
  right: 0;
  top: calc(50% - .5em);
  pointer-events: none;
  color: rgba(0,0,0,0.54);
}

.c1.Mui-disabled {
  color: rgba(0,0,0,0.26);
}

@media (hover:none) {

}

<svg
              aria-hidden={true}
              className="c0 MuiSvgIcon-root MuiSvgIcon-fontSizeMedium c1 MuiSelect-icon MuiSelect-iconStandard"
              data-testid="ArrowDropDownIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>,
            "",
          ]
        </div>,
      ]
    </div>
  </div>,
  "",
]
`;
